src/scrapers/leumi-card.js

```diff
@@ -288,12 +288,89 @@ async function fetchTransactions(browser, options) {
   return allResults;
 }
 
+async function fetchAccountsData(browser) {
+  function parseFloatValue(value) {
+    return parseFloat(value.replace(',', ''));
+  }
+
+  const homePageUrl = `${BASE_URL}/Registred/HomePage.aspx`;
+  const page = await browser.newPage();
+  await page.goto(homePageUrl);
+
+  const results = {};
+  await clickButton(page, 'a#PlaceHolderMain_HomePage1_HomePageTop1_lnkShowListDisplay');
+  await page.waitForNavigation({ waitUntil: 'load' });
```

Comment:
Hi @chilikla out of curiosity why did you use this approach instead of 'waitForNavigationAndDomLoad' or 'waitForRedirect'?